src/app/logic/localStorage.js

```diff
@@ -0,0 +1,23 @@
+function addResult(rankings, gameInfo, amountOfResults) {
+  const { name, result, mode } = gameInfo;
+  // push gameinfo (only name and result, we don't need mode in this place) into the right mode in
+  // rankings
+  rankings[mode].push({ name, result });
```

Comment:
As I understand it, this function is modifying the rankings object of the data from model? This should not be the case. You may want to add methods to the model that will accept its modifications, like Model.addResultToRanking, and perhaps incorporated in it could be local storage service, that would have methods like LocalStorageService.checkLocalStorage, LocalStorageService.addItemToLocalStorage. But also the local storage methods are really small, so its not a must to create separete methods for it. You are enforcing OOP though, so you might want to consider it.